meta-resin-common/recipes-connectivity/dhcp/dhcp_%.bbappend

```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
+
+SRC_URI_append = " \
+    file://dhclient.conf \
+    "
```

Comment:
You can use just one liner here as you file just one file. Like:

SRC_URI_append = " file://dhclient.conf"